mlir/lib/Dialect/MQTOpt/Transforms/MQTCoreRoundTrip.cpp

```diff
@@ -40,7 +39,7 @@ struct MQTCoreRoundTrip final : impl::MQTCoreRoundTripBase<MQTCoreRoundTrip> {
             // This was deprecated in LLVM@20, but the alternative does not yet
             // exist in LLVM@19.
             // NOLINTNEXTLINE(clang-diagnostic-deprecated-declarations)
-            mlir::applyPatternsGreedily(op, std::move(patterns)))) {
+            APPLY_PATTERNS_GREEDILY(op, std::move(patterns)))) {
```

Comment:
The `#include <mlir/Support/LLVM.h>` include was erroneously removed from this file.